CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # CHANGELOG
 
+# Version 3.2.0-BETA.2
+* There were lots of refactorings.
```

Comment:
@trask we might need to reword these in the github release notes.

Reply:
we only need to put user facing changes in the change log
```suggestion
* Lots of internal clean up.
```